frontend/src/components/ScreenSharingButton/ScreenSharingButton.tsx

```diff
@@ -52,7 +55,7 @@ const ScreenSharingButton = ({
 
   return (
     !isMobile() && (
```

Comment:
do we still need this `!isMobile` here?

Reply:
Yes, screensharing is not supported for mobile devices. I'll add a comment. See [link](https://developer.mozilla.org/en-US/docs/Web/API/MediaDevices/getDisplayMedia#browser_compatibility)

Reply:
ahhh that's right. thanks 🙏